submissions/james-shane/scripts/article.js

```diff
@@ -0,0 +1,57 @@
+var articles = [];
+
+function Article (opts) {
+  // TODO: Use the js object passed in to complete this contructor function:
+  // Save ALL the properties of `opts` into `this`.
+  this.title = opts.title;
+  this.category = opts.category;
+  this.author = opts.author;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
```

Comment:
😃  We will soon cover a dynamic approach to capturing **all** properties of an object.